MapboxDirections/MBRouteOptions.swift

```diff
@@ -458,6 +458,26 @@ open class RouteOptions: NSObject, NSSecureCoding {
         }
         return (waypoints, routes)
     }
+    
+    // MARK: NSCopying
+    /**
+     Returns a copy of the object.
+     - parameter zone: an NSZone to use in the copy operation.
+     - returns: A copy of the object.
```

Comment:
I _think_ this method already inherits documentation from the protocol’s method declaration, at least as far as Quick Help and other tools are concerned.